<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameUniverse</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(2, 3, 68, 1), stop:1 rgba(40, 184, 213, 1));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_git">
          <property name="styleSheet">
           <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 14pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
          </property>
          <property name="text">
           <string>GiT</string>
          </property>
          <property name="icon">
           <iconset resource="res-rc.qrc">
            <normaloff>:/icon/logo_icons/git-repository.svg</normaloff>:/icon/logo_icons/git-repository.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_settings">
          <property name="styleSheet">
           <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 14pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="icon">
           <iconset resource="res-rc.qrc">
            <normaloff>:/icon/logo_icons/setting.svg</normaloff>:/icon/logo_icons/setting.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 16pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string> SNAKE</string>
        </property>
        <property name="icon">
         <iconset resource="res-rc.qrc">
          <normaloff>:/icon/logo_icons/snake_logo.svg</normaloff>:/icon/logo_icons/snake_logo.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 16pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string> PONG</string>
        </property>
        <property name="icon">
         <iconset resource="res-rc.qrc">
          <normaloff>:/icon/logo_icons/pong_logo.svg</normaloff>:/icon/logo_icons/pong_logo.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 16pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string> CONTAINER LEAP</string>
        </property>
        <property name="icon">
         <iconset resource="res-rc.qrc">
          <normaloff>:/icon/logo_icons/container.svg</normaloff>:/icon/logo_icons/container.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 16pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string> PAC-MAN</string>
        </property>
        <property name="icon">
         <iconset resource="res-rc.qrc">
          <normaloff>:/icon/logo_icons/pacman.svg</normaloff>:/icon/logo_icons/pacman.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: white;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
width: 230px;
height: 50px;
font-family: Tahoma, sans-serif;
font-weight: bold;
font-size: 16pt
}

QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}</string>
        </property>
        <property name="text">
         <string> DONKEY KONG</string>
        </property>
        <property name="icon">
         <iconset resource="res-rc.qrc">
          <normaloff>:/icon/logo_icons/barrel.svg</normaloff>:/icon/logo_icons/barrel.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res-rc.qrc"/>
 </resources>
 <connections/>
</ui>
